bok_choy/web_app_test.py

```diff
@@ -86,6 +93,63 @@ def unique_id(self):
         """
         return str(uuid4().int)
 
+    @property
+    def axs_audit(self):
+        """
+        Use Google's Accessibility Developer Tools to audit the
+        page for accessibility problems.
+
+        See https://github.com/GoogleChrome/accessibility-developer-tools
+
+        Since this needs to inject JavaScript into the browser page, the only
+        way I know how to do this is if you are using PhantomJS as your browser.
+
+        Raises:
+            NotImplementedError if you are not using PhantomJS
+
+        Returns:
+            A list of objects in the following form, one for each browser session:
+            {
+              /** @type {axs.constants.AuditResult} */
+              result,  // one of PASS, FAIL or NA
+
+              /** @type {Array.<Element>} */
+              elements,  // The elements which the rule fails on,
+                         // if result == axs.constants.AuditResult.FAIL
+
+              /** @type {axs.AuditRule} */
+              rule  // The rule which this result is for.
+            }
+        """
+        if self.browser.name is not 'phantomjs':
+            raise NotImplementedError('Only supported in PhantomJS')
```

Comment:
nit: just a bit more detail in the error message. Like "accessibility audit only supported..."